javaparser-symbol-solver-testing/src/test/test_sourcecode/javasymbolsolver_0_6_0/src/java-symbol-solver-model/com/github/javaparser/symbolsolver/model/declarations/TypeParameterDeclaration.java

```diff
@@ -23,200 +23,173 @@
 import java.util.Optional;
 
 /**
- * Declaration of a type parameter.
- * For example:
- * <p>
- * class A&lt;E extends String&gt;{}
- * </p>
- * <p>
- * In this case <b>E</b> would be a type parameter.
+ * Declaration of a type parameter. For example:
+ *
+ * <p>class A&lt;E extends String&gt;{}
+ *
+ * <p>In this case <b>E</b> would be a type parameter.
  *
  * @author Federico Tomassetti
  */
 public interface TypeParameterDeclaration extends TypeDeclaration {
 
-    /**
-     * Instantiate a TypeParameter defined on a Type with the given data.
-     */
-    static TypeParameterDeclaration onType(final String name, String classQName, List<Bound> bounds) {
-        return new TypeParameterDeclaration() {
-            @Override
-            public String getName() {
-                return name;
-            }
-
-            @Override
-            public boolean declaredOnType() {
-                return true;
-            }
-
-            @Override
-            public boolean declaredOnMethod() {
-                return false;
-            }
-
-            @Override
-            public boolean declaredOnConstructor() {
-                return false;
-            }
-
-            @Override
-            public String getContainerQualifiedName() {
-                return classQName;
-            }
-
-            @Override
-            public String getContainerId() {
-                return classQName;
-            }
-            
-            @Override
-            public TypeParametrizable getContainer() {
-                return null;
-            }
-
-            @Override
-            public List<Bound> getBounds(TypeSolver typeSolver) {
-                return bounds;
-            }
-
-            @Override
-            public String toString() {
-                return "TypeParameter onType " + name;
-            }
-
-            @Override
-            public Optional<ReferenceTypeDeclaration> containerType() {
-                throw new UnsupportedOperationException();
-            }
-        };
+  /** Instantiate a TypeParameter defined on a Type with the given data. */
+  static TypeParameterDeclaration onType(final String name, String classQName, List<Bound> bounds) {
+    return new TypeParameterDeclaration() {
+      @Override
+      public String getName() {
```

Comment:
What has been edited here? 

If it is just changes to indentation, please undo these changes.